spring-session/src/main/java/org/springframework/session/jdbc/JdbcOperationsSessionRepository.java

```diff
@@ -355,6 +360,8 @@ private static GenericConversionService createDefaultConversionService() {
 
 		private final ExpiringSession delegate;
 
+		private long lastAccessedTime;
```

Comment:
Ahh yes, we can. I'll change the ```RowMapper``` to update the deserialized delegate with ```lastAccessedTime``` from column. It will also minimize the impact in other places.